controller/controller.js

```diff
@@ -42,6 +42,29 @@ const getTopics = (request, response) => {
     }).catch(next)
   }
 
+  const postComments = (req, res, next) => {
+    const newComment = req.body;
+    console.log(newComment, "<<<< new comment")
+    insertComment(newComment).then((comment) => {
+        console.log(comment, "<<<<<<<<<<")
```

Comment:
Removing console.logs. A nice and quick way to find all of these would be to use the search functionality of vc code